x-pack/functionbeat/docs/page_header.html

```diff
@@ -0,0 +1,3 @@
+Functionbeat will reach End of Support on October 18, 2023. You should consider
+moving your deployments to the more versatile and efficient Elastic Serverless
+Forwarder.
```

Comment:
What about turning **Elastic Serverless Forwarder** into a link to the docs?

Reply:
@bmorelli25 I thought about that, but decided against it because AFAIK we cannot provide versioned links in the banner files. Do you know if we can?

Reply:
Correct. We can't. In the old APM Server reference, we use the banner to link to `current`. I think that actually works well as we're linking to the index of `current`, not a specific page.

Reply:
Hmm. But I guess you're working with two similarly versioned books?

Reply:
I think it's best to wait on adding the link since the ESF docs are in flight right now. We can come back and add the link later.